content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md

```diff
@@ -0,0 +1,53 @@
+---
+template: "post"
+slug: "the-5-best-exercises-to-build-muscles"
+
+title: "The 5 Best Exerciese-to-build-muscles"
+author: "Soumyadeep Dubey"
+
+cover: "./banner.jpg"
```

Comment:
banner is not showing on the landing page due wrong path should be `./images/banner.jpg`, but eventually you have to remove this as `images/` directory is not required now.
![image](https://user-images.githubusercontent.com/31156696/127346816-9e6445df-80ee-46c8-b58d-fab0258e3327.png)